ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -64,10 +65,15 @@ const EditGasToolTip = ({
   return (
     <div className="edit-gas-tooltip__container">
       {priorityLevel !== PRIORITY_LEVELS.CUSTOM &&
```

Comment:
Maybe not necessary for this PR, but I'm wondering if it's worth it at some point to extract these conditionals to a variable or refactor this in some way. It seems like a lot is going on here. Just a thought.